app/src/main/java/com/example/newsapp/data/entity/News.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.newsapp.data.entity
+
```

Comment:
Let's use Dto suffix for the response entities,
Dbo for the database entities,
no suffix for the entities consumed for UI